cmd/tetra/debug/progs.go

```diff
@@ -96,7 +138,7 @@ Examples:
 	flags := cmd.Flags()
 	flags.BoolVar(&cfg.all, "all", false, "Get all programs")
 	flags.StringVar(&cfg.lib, "bpf-lib", "bpf/objs/", "Location of Tetragon libs (btf and bpf files)")
-	flags.StringVar(&cfg.bpffs, "bpf-dir", "/sys/fs/bpf/tetragon", "Location of bpffs tetragon directory")
+	flags.StringVar(&cfg.bpffs, "bpf-dir", "detect", "Location of bpffs tetragon directory")
```

Comment:
How instead of "detect", we use ""? Would be nice to also update the help text with something like "if left empty, Tetragon will try to auto-detect".

edit: same for bpf-lib

Reply:
ok